File: back-spring/src/main/java/com/pjt1/demo/model/service/CityService.java
package com.pjt1.demo.model.service;

import java.util.List;

import com.pjt1.demo.model.dto.City;


public interface CityService {
	
	public City search(int city_no);
	public List<City> searchAll();
	public void insert(City City);
	public void update(City City);
	public void delete(int city_no);
}

File: front-vue/src/services/CityService.js
import Api from "./Api";

class CityService {
  searchAll() {
    return Api.get("/City")
      .then(res => {
        
        return res.data.data;
      })
      .catch(e => console.log(e));
  }
  search(no) {
    return Api.get(`/City/${no}`)
      .then(res => {
  
        return res.data.data;
      })
      .catch(e => console.log(e));
  }
 
}

export default new CityService();
